addon/src/lib/reducers/sideEffects.js

```diff
@@ -181,6 +183,20 @@ export function reducer(
         installManager.syncInstalled();
       };
 
+    case actions.BROWSER_STARTUP.type:
```

Comment:
Yup, I moved that GA ping over here because it seemed like a lot of code in an otherwise pretty high-level main function. Is there a guideline around when and when not to define actions? I was just using them like side-effecting pubsub events. I can move the code back if you'd rather.